pages/k8s/1.29/upgrading.md

```diff
@@ -0,0 +1,396 @@
+your CNI (e.g. flannel) will cause a small amount of network downtime. Upgrading
+<strong>easyrsa</strong> will not cause any downtime. The behaviour of other
+components you have added to your cluster may vary - check individual documentation
+for these charms for more information on upgrades.
+  </p>
+</div>
+
+## Upgrading Kubernetes
+
+Before you upgrade the **Kubernetes** components, you should be aware of the exact
+release you wish to upgrade to.
+
+The **Kubernetes** charms use **snap** _channels_ to manage the version of
+**Kubernetes** to use. Channels are explained in more detail in the
+[official snap documentation][snap-channels], but in terms of **Kubernetes** all you
+need to know are the major and minor version numbers and the 'risk-level':
+
+| Risk level | Description                                               |
+| ---------- | --------------------------------------------------------- |
+| stable     | The latest stable released version of Kubernetes          |
+| candidate  | Release candidate versions of Kubernetes                  |
+| beta       | Latest alpha/beta of Kubernetes for the specified release |
+| edge       | Nightly builds of the specified release of Kubernetes     |
+
+For most use cases, it is strongly recommended to use the 'stable' version of charms.
+
+### Upgrading the **kube-api-loadbalancer**
+
+A core part of **Charmed Kubernetes** is the kubeapi-load-balancer component. To ensure API service
+continuity this upgrade should precede any upgrades to the **Kubernetes** master and
+worker units.
+
+```bash
+juju refresh kubeapi-load-balancer --channel=1.29/stable
+```
+
+The load balancer itself is based on NGINX, and the version reported by `juju status` is
+that of NGINX rather than Kubernetes. Unlike the other Kubernetes components, there
+is no need to set a specific channel or version for this charm.
+
+### Upgrading the **kubernetes-control-plane** units
+
+**Note**: Older versions of Charmed-Kubernetes used `kubernetes-master` as the charm name. This has been updated
+to `kubernetes-control-plane`. However, it is not possible to rename a deployed application. If you 
+originally installed version 1.23 or before, your units will follow the old naming scheme and you should
+substitute `kubernetes-control-plane`for `kubernetes-master`in the following commands.
+
+To start upgrading the Kubernetes master units, first upgrade the charm:
+
+```bash
+juju refresh kubernetes-control-plane --channel=1.29/stable
+```
+
+Once the charm has been upgraded, it can be configured to select the desired **Kubernetes** channel, which takes the form `Major.Minor/risk-level`. This is then passed as a configuration option to the charm. So, for example, to select the stable 1.29 version of **Kubernetes**, you would enter:
+
+```bash
+juju config kubernetes-control-plane channel=1.29/stable
+```
+
+If you wanted to try a release candidate for 1.26, the channel would be `1.26/candidate`.
```

Comment:
```suggestion
If you wanted to try a release candidate for 1.30, the channel would be `1.30/candidate`.
```
:crossed_fingers:

Reply:
imma stick with 1.29/candidate since 1.29+ck1 will be landing there next.  1.30 is still a ways off.